hugegraph-api/src/main/java/com/baidu/hugegraph/serializer/JsonSerializer.java

```diff
@@ -51,23 +50,16 @@
 
 public class JsonSerializer implements Serializer {
 
-    private final GraphSONWriter writer;
-
     private static final int BUF_SIZE = 128;
     private static final int LBUF_SIZE = 1024;
 
-    public JsonSerializer(GraphSONWriter writer) {
-        this.writer = writer;
+    private static JsonSerializer INSTANCE = new JsonSerializer();
```

Comment:
BUF_SIZE and LBUF_SIZE are unused any more?

Reply:
LBUF_SIZE still in using